### DISCUSSION TITLE
Miller index of 'uvw' unrecogrnized

### DISCUSSION THREAD

**Discussion by amirtopaz** [mtex-toolbox/mtex#1230] [Ask Anything] [answered]
Hi, 
Lately I get a conversion to 'hkl' when I try to add 'uvw' to the Miller index 
I use the command:
h_hbl = [Miller(1,0,0,CS{2},'hkl'), Miller(0,1,0,CS{2},'hkl'),Miller(0,0,1,CS{2},'uvw')];

and I get the following warning:

Warning: Miller indices are converted to hkl 
> In Miller/cat (line 21)
  In vector3d/horzcat (line 4)
  In MTEX_loadEBSD_Plag_Amph (line 189) 

The conversion to 'hkl' shows me a plane called (13.6028.2).
Though the orientations seem as the should look for [001], I want to make sure that this conversion does not distort them somehow.
And also find a way to cancel this auto conversion

This issue is repetitive both in Mtex 5.6 and 5.7

Cheers,
Amir

**Comment by ralfHielscher** [comment, marked as answer]
Hi Amir,

the warning concerns only the display of the Miller indices. The direction is not changed at all.
There is no impact on any computations.

Ralf.

**Reply by amirtopaz** [reply]
Thanks Ralf,
Do you know why it happened?
It didn't use to do it in the past

**Comment by ralfHielscher** [comment]
It always happens if you mix 'uvw' and 'hkl' indices in one list. It did so before as well.

Ralf.
